dataclasses_avroschema/fields.py

```diff
@@ -460,37 +463,52 @@ def fake(self) -> bytes:
 
 @dataclasses.dataclass
 class EnumField(BaseField):
+    _VALID_META_FIELDS = ["aliases", "default", "doc", "namespace"]
+
+    def _get_meta_class_attributes(self) -> typing.Dict[str, typing.Any]:
+        meta_dict = {}
+        if "Meta" in self.type.__members__:
+            meta = self.type.__members__["Meta"].value
+            meta_dict = {k: v for k, v in vars(meta).items() if k in self._VALID_META_FIELDS}
+
+        default = meta_dict.get("default")
+        if default:
+            try:
+                self.type(default)
+            except ValueError as e:
+                raise ValueError(f"Default enum symbol must be one of {self.get_symbols()}", e)
+
+        return meta_dict
+
+    def get_symbols(self) -> typing.List[str]:
+        return [e.value for e in self.type if e.name != "Meta"]
+
     def get_avro_type(self) -> typing.Dict[str, typing.Any]:
+
         avro_type = {
             "type": ENUM,
-            "name": self.get_singular_name(self.name),
-            "symbols": self.default.symbols,
+            "name": self.type.__name__,
+            "symbols": self.get_symbols(),
+            **self._get_meta_class_attributes(),
         }
 
-        if self.default.namespace is not None:
-            avro_type["namespace"] = self.default.namespace
-
-        if self.default.aliases is not None:
-            avro_type["aliases"] = self.default.aliases
-
         return avro_type
 
     def get_default_value(self) -> typing.Union[str, dataclasses._MISSING_TYPE, None]:
-        default = self.default.default
 
-        if default == types.MissingSentinel:
+        if self.default == types.MissingSentinel:
             return dataclasses.MISSING
-        elif default in (dataclasses.MISSING, None):
-            return default
+        elif self.default in (dataclasses.MISSING, None):
+            return self.default
         else:
             # check that the default value is listed in symbols
             assert (
-                default in self.default.symbols
-            ), f"The default value should be on of {self.default.symbols}. Current is {default}"
-            return default
+                self.default.value in self.get_symbols()
+            ), f"The default value should be one of {self.get_symbols()}. Current is {self.default}"
```

Comment:
Consider a priettier join.
```suggestion
            ), f"The default value should be one of {', '.join(self.get_symbols())}. Current is {self.default}"
```